specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-06-01/managedClusters.json

```diff
@@ -5750,6 +5750,55 @@
         ]
       },
       "description": "The type of a snapshot. The default is NodePool."
+    },
+    "GuardrailsProfile": {
```

Comment:
So I see that this change was discussed and approved for `06-02-preveiw`, not `06-01` like you're targeting here. Also, the `06` APIs have already shipped, the Swagger cutoff was `07/03`.

Was your change in the RP but not in the Swagger? AFAIK this change should be targeted at the dev branch (which hasn't been created yet): https://github.com/Azure/azure-rest-api-specs/tree/dev-containerservice-Microsoft.ContainerService-2022-08-02-preview

@FumingZhang will know when it is planned to be created. Probably sometime around 8/19 as per the schedule.

Reply:
Yes, I'll ping you in the api channel after the dev branch for 08-02-preview is created @rsamigullin